.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+<!-- Thanks for submitting this pull request for KeY -->
+<!-- Since the project has a strict review policy, please make the -->
+<!-- reviewer's job easier by providing the necessary information -->
+<!-- in the text below. The comments may remain since they will be -->
+<!-- invisible when showing the PR. -->
```

Comment:
```suggestion
<!--
Thanks for submitting this pull request for KeY.
Since the project has a strict review policy, please make the
reviewer's job easier by providing the necessary information
in the text below. The comments may remain since they will be
invisible when showing the PR.
-->
```
Easier to read (same content). Should ideally be consistent across the template.